src/components/Quill/resources/generateWebViewIndex.ts

```diff
@@ -38,7 +47,9 @@ export function generateWebViewIndex(
         </style>
       </head>
       <body>
-        <div id="editor" class="quill-editor"></div>
+        <div class="ql-wrapper">
```

Comment:
```suggestion
        <div class="quill-wrapper">
```

Dit is consisten met de `quill-editor` die we al hebben.